hack/prune-libraries.sh

```diff
@@ -19,6 +19,8 @@
 
 # By default just checks for extra libraries, run with --fix to make changes.
 
+# TODO: rewrite this in Python, or fix this!
+# shellcheck disable=SC2086,SC2046
```

Comment:
I have no idea what this means, nor whether/when it is safe to remove them.

Can you disable the lines that violate these rather than the whole file?

Reply:
Bump